README.md

```diff
@@ -71,30 +72,51 @@ Please refer to [changelog.md](docs/en/changelog.md) for details and release his
 
 ## Benchmark and model zoo
 
-Results and models are available in the [model zoo](docs/en/model_zoo.md).
+Results and models are available in the [model zoo](docs/en/model_zoo.md). The supported (:white_check_mark:) and on-the-way (:o:) methods and datasets are listed below.
 
-Supported methods of video object detection:
-
-- [x] [DFF](configs/vid/dff) (CVPR 2017)
-- [x] [FGFA](configs/vid/fgfa) (ICCV 2017)
-- [x] [SELSA](configs/vid/selsa) (ICCV 2019)
-- [x] [Temporal RoI Align](configs/vid/temporal_roi_align) (AAAI 2021)
-
-Supported methods of multi object tracking:
-
-- [x] [SORT/DeepSORT](configs/mot/deepsort) (ICIP 2016/2017)
-- [x] [Tracktor](configs/mot/tracktor) (ICCV 2019)
-- [x] [QDTrack](configs/mot/qdtrack) (CVPR 2021)
-- [x] [ByteTrack](configs/mot/bytetrack) (arXiv 2021)
-
-Supported methods of single object tracking:
-
-- [x] [SiameseRPN++](configs/sot/siamese_rpn) (CVPR 2019)
-- [x] [STARK](configs/sot/stark) (ICCV 2021)
-
-Supported methods of video instance segmentation:
-
-- [x] [MaskTrack R-CNN](configs/vis/masktrack_rcnn) (ICCV 2019)
+<style>
+td, th {
+   border: none!important;
+}
+</style>
+
+### Video Object Detection
+
+| Method                                                                              | Dataset                                                                  |
+| ----------------------------------------------------------------------------------- | ------------------------------------------------------------------------ |
+| :white_check_mark:  [DFF](configs/vid/dff) (CVPR 2017)                              | :white_check_mark: [ILSVRC](http://image-net.org/challenges/LSVRC/2017/) |
```

Comment:
I think we could use `[x]` and `[ ]` to indicate the supported and on the way methods. 
`white_check_mark` and `:o:` seem a little ugly.

Reply:
I agree. But github readme does not support use [x] and [] in a table.

Reply:
Could you provide any better solution?

Reply:
Maybe using the following structure?
```
### Video Object Detection

Supported Methods
- [x] aaa
- [x] aaa
- [ ] bbb

Supported Datasets
- [x] aaa
- [x] aaa
```

Reply:
Good idea. Have adjusted it in the new commit.